test/functional/ex_cmds/oldfiles_spec.lua

```diff
@@ -1,49 +1,92 @@
-local h = require('test.functional.helpers')
+local Helpers = require('test.functional.helpers')
+local Screen  = require('test.functional.ui.screen')
 
-local buf     = h.curbufmeths
-local command = h.command
-local eq      = h.eq
-local execute = h.execute
-local feed    = h.feed
-local nvim    = h.nvim
+local buf         = Helpers.curbufmeths
+local command     = Helpers.command
+local eq          = Helpers.eq
+local execute     = Helpers.execute
+local feed        = Helpers.feed
+local nvim        = Helpers.nvim
+local nvim_prog   = Helpers.nvim_prog
+local set_session = Helpers.set_session
+local spawn       = Helpers.spawn
 
 local shada_file = 'test.shada'
 
--- h.clear() uses "-i NONE", which is not useful for this test.
-local function clear()
+--
+-- Helpers.clear() uses "-i NONE", which is not useful for this test.
+--
+local function reset()
```

Comment:
prefixing it with a underscore `_clear()`) would also be fine, to show that the semantics are the same, but the scope and implementation are different.